packages/manager/cypress/e2e/core/longview/longview.spec.ts

```diff
@@ -100,85 +73,133 @@ const waitForLongviewData = (
-
-      interceptGetLongviewClients().as('getLongviewClients');
-      interceptFetchLongviewStatus().as('fetchLongviewStatus');
-      cy.visitWithLogin('/longview');
-      cy.wait('@getLongviewClients');
-
-      // Find the table row for the new Longview client, assert expected information
-      // is displayed inside of it.
-      cy.get(`[data-qa-longview-client="${client.id}"]`)
-        .should('be.visible')
-        .within(() => {
-          cy.findByText(client.label).should('be.visible');
-          cy.findByText(client.api_key).should('be.visible');
-          cy.contains(installCommand).should('be.visible');
-          cy.findByText('Waiting for data...');
-        });
-
-      // Install Longview on Linode by SSHing into machine and executing cURL command.
-      installLongview(linodeIp, linodePassword, installCommand);
-
-      // Wait for Longview to begin serving data and confirm that Cloud Manager
-      // UI updates accordingly.
-      waitForLongviewData('fetchLongviewStatus', client.api_key);
-
-      // Sometimes Cloud Manager UI does not updated automatically upon receiving
-      // Longivew status data. Performing a page reload mitigates this issue.
-      // TODO Remove call to `cy.reload()`.
-      cy.reload();
-      cy.get(`[data-qa-longview-client="${client.id}"]`)
-        .should('be.visible')
-        .within(() => {
-          cy.findByText('Waiting for data...').should('not.exist');
-          cy.findByText('CPU').should('be.visible');
-          cy.findByText('RAM').should('be.visible');
-          cy.findByText('Swap').should('be.visible');
-          cy.findByText('Load').should('be.visible');
-          cy.findByText('Network').should('be.visible');
-          cy.findByText('Storage').should('be.visible');
-        });
+    mockGetLongviewClients([client]).as('getLongviewClients');
+    mockFetchLongviewStatus(client, 'lastUpdated', longviewLastUpdatedWaiting);
+    mockFetchLongviewStatus(client, 'getValues', longviewGetValuesWaiting);
+    mockFetchLongviewStatus(
+      client,
+      'getLatestValue',
+      longviewGetLatestValueWaiting
+    ).as('fetchLongview');
+
+    const installCommand = getInstallCommand(client.install_code);
+
+    cy.visitWithLogin('/longview');
+    cy.wait('@getLongviewClients');
+
+    // Confirm that Longview landing page lists a client that is still waiting for data...
+    cy.get(`[data-qa-longview-client="${client.id}"]`)
+      .should('be.visible')
+      .within(() => {
+        cy.findByText(client.label).should('be.visible');
+        cy.findByText(client.api_key).should('be.visible');
+        cy.contains(installCommand).should('be.visible');
+        cy.findByText('Waiting for data...');
+      });
+
+    // Update mocks after initial Longview fetch to simulate client installation and data retrieval.
+    // The next time Cloud makes a request to the fetch endpoint, data will start being returned.
+    cy.wait(['@fetchLongview']).then(() => {
```

Comment:
```suggestion
    cy.wait(['@fetchLongview', '@fetchLongview', '@fetchLongview']).then(() => {
```
We do want to revert both of these back to the way they were -- we need to wait for all 3 requests to resolve before proceeding to ensure that each response is mocked with the correct data.